packages/kiota_serialization_text/test/text_parse_node_test.dart

```diff
@@ -88,7 +88,7 @@ void main() {
 
       expect(
         node.getGuidValue(),
-        equals(UuidValue.nil),
+        equals(Namespace.nil),
```

Comment:
This seems to pass again if i change it to 
`equals(UuidValue.raw(Namespace.nil.value)),
`

Reply:
Thanks, didn't realize at the time that this didn't compile. Change effectively reverted